exporter/exporter_test.go

```diff
@@ -277,6 +278,14 @@ func TestConnectionDurations(t *testing.T) {
 	}
 }
 
+// TODO Rework keys clean up.
+//      Clean all on startup or force `docker-compose down && up` each `make docker-test`
```

Comment:
I wasn't sure if I had the right to edit the Makefile myself if this case.

Reply:
Yup, you can edit the make file. 
You can handle this problem in two ways.
Either, you assume that the test Redis DB is always empty and clean to start with. 
Or you can run quick count before kicking off the real test and then you know what to expect and existing keys aren't a problem.
I don't have a strong preference for either way. In the end, the Drone CI test is what matters for PRs and both options will work for that.
Also, this is useful information and should either go to the top of the file or into a "Development" section in the README so it's easier to find.